test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
```

Comment:
By default generating a resources comes up with spec/fixture files. IF you don't plan to use it right away for test writing, please remove such files, as they give false impression, that you actually test your code ;-) Furthermore, my plan is to use factory_girl + rspec, so these are obsolete anyway